pkg/services/apiserver/README.md

```diff
@@ -79,3 +79,14 @@ The Kubernetes compatible API can be accessed using existing Grafana AuthN at: [
 
 The equivalent openapi docs can be seen in [http://localhost:3000/swagger](http://localhost:3000/swagger), 
 select the relevant API from the dropdown in the upper right.
+
+## General Structure
+
```

Comment:
```suggestion
The folder structure aims to follow the patterns established in standard (https://github.com/kubernetes/kubernetes)[kubernetes] projects.
```

Reply:
good call. the original info made more sense in the context of the conversation i had with @leonorfmartins, but without that it was very unclear. i added some additional info about how the folders are structured in [5f62507](https://github.com/grafana/grafana/pull/87508/commits/5f6250745c6a0393e919de7e8b5ae4b68911c680)